src/components/PositionsList/PositionsList.tsx

```diff
@@ -63,6 +86,13 @@ export const PositionsList: React.FC<IProp> = ({
           </Typography>
         )}
       </Grid>
+      {paginator(data, page, itemsPerPage).totalPages > 1 ? (
```

Comment:
maybe just use "Math.ceil(data.length / itemsPerPage)" instead of calling paginator multiple times